pjsip/src/pjsua-lib/pjsua_call.c

```diff
@@ -1625,8 +1684,8 @@ pj_bool_t pjsua_call_on_incoming(pjsip_rx_data *rdata)
     } else {
         acc_id = call->acc_id = pjsua_acc_find_for_incoming(rdata);
         if (acc_id == PJSUA_INVALID_ID) {
-            pjsip_endpt_respond_stateless(pjsua_var.endpt, rdata,
-                                          PJSIP_SC_TEMPORARILY_UNAVAILABLE,
+            ret_st_code = PJSIP_SC_TEMPORARILY_UNAVAILABLE;
```

Comment:
Here you are using enums, in other files integers. Is there a reason for different approaches?

Reply:
Usually we avoid unnecessary changes, however with the use of the new var we can also change the status code to using the enum.